src/main/kotlin/com/terraformation/backend/config/TerrawareServerConfig.kt

```diff
@@ -117,10 +117,12 @@ class TerrawareServerConfig(
       val enabled: Boolean = true,
 
       /**
-       * What time of day the daily tasks are run. This is treated as a local time in the configured
-       * [timeZone]. Default is 8AM.
+       * What time of day the daily tasks are run. For server-wide tasks, the globally-configured
+       * server time zone ([timeZone]) is used. For tasks related to a particular facility,
+       * organization, etc., that entity's time zone is used. Default is 00:01 (1 minute past
```

Comment:
Not an issue since we check every 15minutes anyways - just calling out that technically 12:00am is start of day.. if that was the intent.

Reply:
No technical reason for this; I just used 12:01 because it's what the product folks were saying in the PRD review. I guess if I wanted to invent a technical reason, I could talk about protecting against clock skew in the cluster or something, but really it's just what people said in the PRD review.

Reply:
Got it, this is totally fine! Yes, I recall 12:01 being picked during the review.